### PR TITLE
Merge Branch-ReadMe-Vinod with Master

### LINKED ISSUE #5: Update README.md

### PR DESCRIPTION
**Pull request by Vinodjayakumar124**
Closes #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
-* The project simulates an ongoing software project for a desktop application (called _AddressBook_) used for managing contact details.
-  * It is **written in OOP fashion**. It provides a **reasonably well-written** code base **bigger** (around 6 KLoC) than what students usually write in beginner-level SE modules, without being overwhelmingly big.
-  * It comes with a **reasonable level of user and developer documentation**.
-* It is named `AddressBook Level 3` (`AB3` for short) because it was initially created as a part of a series of `AddressBook` projects (`Level 1`, `Level 2`, `Level 3` ...).
-* For the detailed documentation of this project, see the **[Address Book Product Website](https://se-education.org/addressbook-level3)**.
-* This project is a **part of the se-education.org** initiative. If you would like to contribute code to this project, see [se-education.org](https://se-education.org#https://se-education.org/#contributing) for more info.
+### Events
+![Ui_events](docs/images/Ui_events.png)
+
+# Duke-The-Market
```
Comment (Benjamin-Sim) [inline README.md]: I believe our product name is "Duke The Market" (without hyphens)

File: README.md [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
+
+Duke-The-Market is a one-stop marketing tool that allows department stores to keep track of their upcoming marketing plan roll outs, monitor its impact, and to target the appropriate subsegment of its customer base for each of those plans.
```
Comment (Benjamin-Sim) [inline README.md]: Same as above

**Review by Benjamin-Sim — COMMENTED** [review]
Cool mockup of the events UI! 👍

For the Persons UI, should we remove the purchase history since we're not adding it anymore?

Also some small issues with the product name.

File: README.md (hunk 2 above)
Comment (Vinodjayakumar124) [reply README.md]: Ok

File: README.md (hunk 1 above)
Comment (Vinodjayakumar124) [reply README.md]: Ok thanks! Have made this change as well as editing the UI to remove purchase history

**Review by Benjamin-Sim — APPROVED** [review]
LGTM! 👍

**Pull request merged**